specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -3718,41 +3666,43 @@
           "description": "Customer consent for enabling AAD pod identity addon in cluster using Kubenet network plugin."
```

Comment:
What is this `allownetworkPluginKubenet` actually enabling for me? Why would I want to enable it?

Reply:
Fixed